networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/AttesterSlashingTopicHandler.java

```diff
@@ -92,7 +81,9 @@ public Bytes4 getForkDigest() {
     return forkDigest;
   }
 
-  protected InternalValidationResult validateData(final AttesterSlashing attesterSlashing) {
-    return validator.validate(attesterSlashing);
+  @Override
+  protected SafeFuture<InternalValidationResult> validateData(
+      final AttesterSlashing attesterSlashing) {
+    return SafeFuture.completedFuture(validator.validate(attesterSlashing));
```

Comment:
Probably outside the scope of this PR but does this wind up doing the validation on the netty thread or have we already jumped off of it?

Because if we've got `SafeFuture` involved anyway it would be good to delegate validation to a worker pool and not block the netty threads.

Reply:
We're still on the netty thread.  Definitely makes sense to move this to a worker thread though 👍  We've got a ticket for that here: https://github.com/PegaSysEng/teku/issues/1947.  Should be made easier by this PR.